src/py42/_internal/clients/archive.py

```diff
@@ -10,6 +11,76 @@ def get_data_key_token(self, device_guid):
         data = {u"computerGuid": device_guid}
         return self._session.post(uri, data=json.dumps(data))
 
+    def get_single_archive(self, archive_guid):
+        """Gets single archive information by GUID.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            archive_guid (int): The GUID for the archive.
+
+        Returns:
+            :class:`py42.response.Py42Response`: A response containing archive information.
+        """
+        uri = u"/api/Archive/{}".format(archive_guid)
+        return self._session.get(uri)
+
+    def get_page(self, page_num, page_size=None, **kwargs):
+        """Gets an individual page of archives.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            page_num (int): The page number to request.
+            page_size (int, optional): The number of archives to return per page.
```

Comment:
Make sure to end optional parameter doc strings with `Defaults to None` (or whatever the default value is),

Reply:
Updated optional parameters to include that.